e2e/cypress/test_module/cypress/support/step_definitions/home-page-general.steps.ts

```diff
@@ -1,7 +1,7 @@
 /// <reference types="cypress" />
 
 import { Given, Then, When } from "@badeball/cypress-cucumber-preprocessor";
-import { homePage } from "../page_objects/home-pages";
+import { homePage } from "../../fixtures/global-data";
```

Comment:
Can we use alias here?

Reply:
> Can we use alias here?

We don't use `aliases` for cypress testing.
Are we have setup config for that ?

Reply:
Yes we have config for aliases. You can find it in webpack.config.js in e2e folder and it was setuped by expert. I have a suggestion:

```suggestion
import { ERRORS, homePage } from "@cypress-e2e/fixtures/global-data";
```

Reply:
> Yes, we have config for aliases. You can find it in `webpack.config.js` in `e2e` folder and it was setuped by expert. I have a suggestion:

thanks. saw it